src/include/logging/checkpoint_manager.h

```diff
@@ -1,69 +0,0 @@
-//===----------------------------------------------------------------------===//
```

Comment:
Is checkpointing included in this project?

Reply:
Yes, but we will take care of it in another PR.